### PR TITLE
Link to conda-forge in Readme

### PR DESCRIPTION
**Pull request by JanakiSheth**
I found it useful to install skorch using conda-forge, and someone else might find it useful  too.

### CODE REVIEW

**Comment by ottonemo** [comment]
Thank you for the pull request, this looks fine in general!

I'm not sure about linking to a 3rd party resource, though. I appreciate when people take the source and distribute skorch packages over some platform like conda-forge but I'm hesitant linking to it in the README since none of the skorch maintainers control the conda-forge repository. So in the worst case the conda-forge repository gets compromised without our knowledge and we encourage users to use the compromised package.

But maybe that's just paranoia. @BenjaminBossan what do you think?

**Comment by BenjaminBossan** [comment]
In theory, it's possible that pytorch gets compromised and we encourage users to install it, so there is no absolute safety. But it would probably not hurt to mention that we are not responsible for the forge repo, since that could be something that users might assume when reading about it in the README.md.

**Comment by BenjaminBossan** [comment]
@JanakiSheth Would you mind adding a note that the conda channel is not managed the skorch maintainers?

**Comment by hmaarrfk** [comment]
Hey, conda-forge contributor here. You can probably just submit the PR (using the github editor) to add yourselves to the bottom of the maintainers list explaining that you are a core dev of skorch. That way you would retain some control over the distribution.

https://github.com/conda-forge/skorch-feedstock/blob/master/recipe/meta.yaml#L47

Seeing as this seems like a pure python package, it should just be OK and minimal maintenance effort as bots do most of the work for pypi/github release packages.

**Comment by BenjaminBossan** [comment]
@hmaarrfk Thx for the hint. I'm not quite sure how this works. Once you are on the maintainer list, you are able to directly push to the repo?

**Comment by hmaarrfk** [comment]
@benjamin-work yeah, conda-forge is basically a giant automation machine. They ask that you not push directly to the repo since it triggers packages to be released. Instead, you work on your fork and make a PR. Once everything is passing on CIs (travis/appveyor/circleci/azure), you can merge it in, and the package gets released to Anaconda automatically.

I think it would be instructive to package an other small project of yours to get the hang of things:
https://github.com/conda-forge/staged-recipes
is the place to start.
Instructions for how to use it are at the bottom of that link.

**Comment by BenjaminBossan** [comment]
@ottonemo Let's make some progress. Would you be fine with the change?

**Comment by ottonemo** [comment]
@thomasjpfan @BenjaminBossan please review :)

File: README.rst [hunk 1]
```diff
@@ -223,15 +235,15 @@ In general, this should work (assuming CUDA 9):
 .. code:: bash
 
     # using conda:
-    conda install pytorch -c pytorch
+    conda install pytorch cudatoolkit=9.0 -c pytorch
```
Comment (thomasjpfan) [inline README.rst]: Do we want users on 10.0?

File: README.rst [hunk 2]
```diff
@@ -12,7 +12,7 @@ A scikit-learn compatible neural network library that wraps PyTorch.
     :scale: 100%
     :target: https://travis-ci.org/dnouri/skorch?branch=master
```
Comment (thomasjpfan) [inline README.rst]:
```rst
.. image:: https://travis-ci.com/skorch-dev/skorch.svg?branch=master
    :target: https://travis-ci.com/skorch-dev/skorch
```

File: README.rst (hunk 1 above)
Comment (ottonemo) [reply README.rst]: PyTorch uses 9 as the default, so we'll stick with 9 for now, I guess.

**Pull request merged**